.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -200,8 +200,9 @@ jobs:
           retention-days: 15
           if-no-files-found: error
 
-  # We don't need the binaries installed in the rootfs as part of the tarball, so can delete them now we've built the rootfs
+  # We don't need the binaries installed in the rootfs as part of the release tarball, so can delete them now we've built the rootfs
   remove-rootfs-binary-artifacts:
+    if: ${{ inputs.stage == 'release' }}
```

Comment:
Nit: how about we rather put this condition in the step (line 215), so that we don't need the always() condition in `build-asset-shim-v2`?

Reply:
That sounds like a much, much better idea. Thanks!